### PR TITLE
fix typo in README

### PR DESCRIPTION
**Pull request by yazdandeli**
Found a small typo in the README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@
 This project contains scripts and patches to build packages for the [Termux]
 Android application.
 
-The packages available here are only from main set. We have some additional
+The pakages available here are only from main set. We have some additional
```
Comment (yazdandeli) [inline README.md]: D

File: README.md [hunk 2]
```diff
@@ -8,7 +8,7 @@
 This project contains scripts and patches to build packages for the [Termux]
 Android application.
 
-The packages available here are only from main set. We have some additional
```
Comment (yazdandeli) [inline README.md]: - [ ]

File: README.md (hunk 1 above)
Comment (yazdandeli) [inline README.md]: N
```suggestion
The pakages available here are only from main set. We have some additional
```

**Comment by Grimler91** [comment]
Thanks!

File: README.md (hunk 1 above)
Comment (yazdandeli) [inline README.md]: Attach files to your comment
Attach files by dragging & dropping, selecting or pasting them.

The

**Comment by xeffyr** [comment]
**How this was merged ???**

**Comment by Grimler91** [comment]
@xeffyr Yeah, good question, I seem to have thought it was the reversed patch. My mistake.

**Pull request merged**